docs/installation/Installation.md

```diff
@@ -63,8 +63,10 @@ You can check example usage at [Nussknacker Quickstart repository](https://githu
 
 All configuration options are described in [Configuration](../installation_configuration_guide/DesignerConfiguration.md).
 
-Some of them can be configured using environment variables, which is mostly helpful in Docker setup.
-In the table below there are all environment variables used in Nussknacker image. $NUSSKNACKER_DIR is a placeholder pointing to Nussknacker installation directory.
+Some of them can be configured using already predefined environment variables, which is mostly helpful in Docker setup.
+In the table below there are all predefined environment variables used in Nussknacker image. $NUSSKNACKER_DIR is a placeholder pointing to Nussknacker installation directory.
+
+Because we use [Typesafe Config](https://github.com/lightbend/config) library, you can set (or override) any configuration value used by Nussknacker even if there is no an already predefined environment variable. This is achieved by setting the JVM property `-Dconfig.override_with_env_vars=true` and setting  environment variables following conventions described [here](https://github.com/lightbend/config?tab=readme-ov-file#optional-system-or-env-variable-overrides).
```

Comment:
1. We should rather focus on [HOCON](https://github.com/lightbend/config/blob/main/HOCON.md) rather than Typesafe Config. The lib is just an implementation of the format (right, it's the most known one, but still). 

2. "setting  environment" - redundant space